src/main/java/net/geforcemods/securitycraft/blocks/reinforced/HorizontalReinforcedIronBars.java

```diff
@@ -0,0 +1,146 @@
+package net.geforcemods.securitycraft.blocks.reinforced;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.IBucketPickupHandler;
+import net.minecraft.block.ILiquidContainer;
+import net.minecraft.block.SlabBlock;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.fluid.Fluids;
+import net.minecraft.fluid.IFluidState;
+import net.minecraft.item.BlockItemUseContext;
+import net.minecraft.item.ItemStack;
+import net.minecraft.pathfinding.PathType;
+import net.minecraft.state.BooleanProperty;
+import net.minecraft.state.StateContainer;
+import net.minecraft.state.properties.BlockStateProperties;
+import net.minecraft.util.BlockRenderLayer;
+import net.minecraft.util.Direction;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.shapes.ISelectionContext;
+import net.minecraft.util.math.shapes.VoxelShape;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.storage.loot.LootContext.Builder;
+
+public class HorizontalReinforcedIronBars extends BaseReinforcedBlock implements IBucketPickupHandler, ILiquidContainer
+{
+	public static final BooleanProperty WATERLOGGED = BlockStateProperties.WATERLOGGED;
+	protected static final VoxelShape SHAPE = Block.makeCuboidShape(-8.0D, 14.0D, -8.0D, 24.0D, 16.0D, 24.0D);
+
+
```

Comment:
Unnecessary newline

Reply:
`When looking at the edges of the horizontal reinforced iron bars, it cannot be selected, allowing to access blocks behind it.`
This is a consequence of having a render model larger than the 1x1x1 standard block size. In fact, you can also place a regular block next to the central "block" of the horizontal bars and textures will collide. Right now I don't know how to provide and easy fix, I guess to be done properly we should create a 3x1x1 block, or create proper connectable horizontal bars. As I said in Discord, I knew this would have limitations, since it's goal/application is very specific.

Reply:
The blocks-placeable-next-to-it-problem isn't really that big of a deal, because as you said it's a very specific use case. As for the selection box, I asked on the Forge discord server and except for placing adjacent blocks it's not possible to fix it. It's on you whether you want to quickly fix it or not. I'm fine with how it is at the moment.

Reply:
If you're fine, I'm fine hahaha. I mean, you're the mod owner after all, you if you feel this is ok to be merged as it is, go for it, and if not, we'll find a way to do things better, no hard feelings :). But as you say, given the situation and the block purpose, I don't really think it's such a big issue